client/src/components/Search/index.js

```diff
@@ -0,0 +1,144 @@
+import React, { Component } from 'react';
+import { Select, Radio, Button } from 'antd';
+import world from 'full-countries-cities';
+import 'antd/dist/antd.css';
+
+import Keywords from '../Keywords';
+
+import './style.css';
+
+const { Option } = Select;
+
+class Search extends Component {
+  state = {
+    country: undefined,
+    city: undefined,
+    lookingFor: '',
+    keyword: '',
+    viewKeywords: false,
+  };
+
+  renderOptions = list =>
+    list.map(item => (
+      <Option key={item} value={item}>
+        {item}
+      </Option>
+    ));
+
+  onSubmit = () => {};
+
+  setKeyword = value => {
+    this.setState({ keyword: value, viewKeywords: false });
+  };
+
+  hideKeyword = () => {
+    this.setState(({ viewKeywords }) => ({ viewKeywords: !viewKeywords }));
+  };
+
+  dropdownFilter = (input, option) =>
+    option.props.children.toLowerCase().indexOf(input.toLowerCase()) >= 0;
+
+  render() {
+    const { country, city, keyword, viewKeywords, lookingFor } = this.state;
+
+    return (
+      <form className="search__form">
+        <div className="country-city-container">
+          <div className="autocomplete-box">
+            <div>WHICH COUNTRY?</div>
+            <Select
+              showSearch
+              placeholder="Select"
+              optionFilterProp="children"
+              onChange={value => this.setState({ country: value, city: '' })}
+              value={country}
+              filterOption={this.dropdownFilter}
+            >
+              {this.renderOptions(world.getCountryNames())}
+            </Select>
+          </div>
+          <div className="autocomplete-box">
+            <div>WHICH CITY?</div>
+            <Select
+              showSearch
+              placeholder="Select"
+              optionFilterProp="children"
+              onChange={value => {
+                this.setState({ city: value });
+              }}
+              value={city}
+              filterOption={this.dropdownFilter}
+            >
+              {country &&
+                world.getCities(country) !== undefined &&
+                this.renderOptions(world.getCities(country))}
```

Comment:
what does `world.getCities()` do? Is it calling some API over the network? It is being called twice here, which you wanna avoid if it's an expensive operation. In such case it should be done at the beginning of the render method (`const cities = world.getCities(ountry);`)